R/tm_g_distribution.R

```diff
@@ -1160,6 +1170,48 @@ srv_distribution <- function(id,
       modal_title = "R Code for distribution",
       code_header = "Distribution"
     )
+
+    ### REPORTER
+    if (with_reporter) {
+      card_fun <- function(comment) {
+        card <- teal.reporter::TealReportCard$new()
+        card$set_name("Association plot")
+        card$append_text("Association plot", "header2")
+        card$append_text("Filter State", "header3")
+        card$append_fs(datasets$get_filter_state())
+        card$append_text("Main Element", "header3")
+        if (input$tabs == "Histogram") {
+          card$append_plot(dist_r(), dim = pws$dim())
+        } else {
+          card$append_plot(qq_r(), dim = pws1$dim())
+        }
```

Comment:
```suggestion
        if (input$tabs == "Histogram") {
          card$append_plot(dist_r(), dim = pws1$dim())
        } else if (input$tabs == "QQplot") {
          card$append_plot(qq_r(), dim = pws2$dim())
        }
```

We persists the `else if` even in the main module code as a tab could be added ...